### PR TITLE
update to  ESCOMP share1.1.5

### PR DESCRIPTION
**Pull request by mvertens**
Verified that this solves compilation problems when cdeps is updated to cdeps1.0.58.

### CODE REVIEW

File: src/nuopc_shr_methods.F90 [hunk 1]
```diff
@@ -18,6 +18,7 @@ module nuopc_shr_methods
   use ESMF         , only : ESMF_Time, ESMF_TimeGet, ESMF_TimeSet, ESMF_ClockGetAlarm
   use ESMF         , only : ESMF_TimeInterval, ESMF_TimeIntervalSet, ESMF_TimeIntervalGet
   use ESMF         , only : ESMF_VM, ESMF_VMGet, ESMF_VMBroadcast, ESMF_VMGetCurrent
+  use ESMF         , only : ESMF_ClockGetNextTime
```
Comment (gold2718) [inline src/nuopc_shr_methods.F90]: Is this used anywhere? Can it be removed?

**Review by gold2718 — CHANGES_REQUESTED** [review]
Question about an import

File: src/nuopc_shr_methods.F90 (hunk 1 above)
Comment (mvertens) [reply src/nuopc_shr_methods.F90]: Good catch. I just removed it.

**Review by gold2718 — APPROVED** [review]
LGTM

**Pull request merged**